### DISCUSSION TITLE
x-show on table tr elements throws errors on click

### DISCUSSION THREAD

**Discussion by zenfinity** [alpinejs/alpine#3877] [7. Bugs (archived)]
I have a table with a category heading row that is clickable and is always showing, and has rows underneath that are shown.
`
<tr class="pa2 mb2 mt2 v-bot category-row-shown" x-on:click="shown_group_1 = !shown_group_1" x-bind:class="shown_group_1 ? 'category-row-shown' : 'category-row'"><td class="pa2 br bw2 b--dark-blue v-mid relative"><span class="dib v-mid pl2 w-80">Protecting Abortion Access</span><span class="dib gray absolute right-2 top-1" x-data="{button: '+'}" x-init="$watch('shown_group_1', value => value ? button = '-' : button = '+' )" x-text="button">-</span></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr x-show="shown_group_1" class="pa2 ma2" style=""><td class="pv2 ph3 v-mid category-column">Clinic safety ordinance</td><td class="pv2 ph3 v-mid city-column"><img src="/src/img/Icons/ScorecardIcons/No.svg" class="w1"></td><td class="pv2 ph3 v-mid city-column"><img src="/src/img/Icons/ScorecardIcons/No.svg" class="w1"></td><td class="pv2 ph3 v-mid city-column"><img src="/src/img/Icons/ScorecardIcons/Yes.svg" class="w1"></td><td class="pv2 ph3 v-mid city-column"><img src="/src/img/Icons/ScorecardIcons/Yes.svg" class="w1"></td><td class="pv2 ph3 v-mid city-column"><img src="/src/img/Icons/ScorecardIcons/No.svg" class="w1"></td><td class="pv2 ph3 v-mid city-column"><img src="/src/img/Icons/ScorecardIcons/No.svg" class="w1"></td><td class="pv2 ph3 v-mid city-column"><img src="/src/img/Icons/ScorecardIcons/No.svg" class="w1"></td><td class="pv2 ph3 v-mid city-column"><img src="/src/img/Icons/ScorecardIcons/Yes.svg" class="w1"></td><td class="pv2 ph3 v-mid city-column"><img src="/src/img/Icons/ScorecardIcons/No.svg" class="w1"></td><td class="pv2 ph3 v-mid city-column"><img src="/src/img/Icons/ScorecardIcons/Yes.svg" class="w1"></td><td class="pv2 ph3 v-mid city-column"><img src="/src/img/Icons/ScorecardIcons/DataNotAvailable.svg" class="w1"></td></tr>
<tr x-show="shown_group_1" class="pa2 ma2" style=""><td class="pv2 ph3 v-mid category-column">Regulations on deceptive practices of anti-abortion pregnancy centers</td><td class="pv2 ph3 v-mid city-column"><img src="/src/img/Icons/ScorecardIcons/No.svg" class="w1"></td><td class="pv2 ph3 v-mid city-column"><img src="/src/img/Icons/ScorecardIcons/No.svg" class="w1"></td><td class="pv2 ph3 v-mid city-column"><img src="/src/img/Icons/ScorecardIcons/No.svg" class="w1"></td><td class="pv2 ph3 v-mid city-column"><img src="/src/img/Icons/ScorecardIcons/No.svg" class="w1"></td><td class="pv2 ph3 v-mid city-column"><img src="/src/img/Icons/ScorecardIcons/No.svg" class="w1"></td><td class="pv2 ph3 v-mid city-column"><img src="/src/img/Icons/ScorecardIcons/Yes.svg" class="w1"></td><td class="pv2 ph3 v-mid city-column"><img src="/src/img/Icons/ScorecardIcons/No.svg" class="w1"></td><td class="pv2 ph3 v-mid city-column"><img src="/src/img/Icons/ScorecardIcons/No.svg" class="w1"></td><td class="pv2 ph3 v-mid city-column"><img src="/src/img/Icons/ScorecardIcons/No.svg" class="w1"></td><td class="pv2 ph3 v-mid city-column"><img src="/src/img/Icons/ScorecardIcons/Yes.svg" class="w1"></td><td class="pv2 ph3 v-mid city-column"><img src="/src/img/Icons/ScorecardIcons/DataNotAvailable.svg" class="w1"></td></tr>
`

Upon click, this error is thrown in the same number as there are hidden rows.
> cdn.min.js:5 Uncaught (in promise) TypeError: u is not a function at cdn.min.js:5:7955

Without x-transition or x-collapse, the rows will not hide again. And with either of them, this is throw individually for every hidden row:

`
<tr x-show="shown_group_1" x-transition.duration.50ms="" x-transition.origin.top="" class="pa2 ma2" style="display: none;"><td class="pv2 ph3 v-mid category-column">Clinic safety ordinance</td><td class="pv2 ph3 v-mid city-column"><img src="/src/img/Icons/ScorecardIcons/No.svg" class="w1"></td><td class="pv2 ph3 v-mid city-column"><img src="/src/img/Icons/ScorecardIcons/No.svg" class="w1"></td><td class="pv2 ph3 v-mid city-column"><img src="/src/img/Icons/ScorecardIcons/Yes.svg" class="w1"></td><td class="pv2 ph3 v-mid city-column"><img src="/src/img/Icons/ScorecardIcons/Yes.svg" class="w1"></td><td class="pv2 ph3 v-mid city-column"><img src="/src/img/Icons/ScorecardIcons/No.svg" class="w1"></td><td class="pv2 ph3 v-mid city-column"><img src="/src/img/Icons/ScorecardIcons/No.svg" class="w1"></td><td class="pv2 ph3 v-mid city-column"><img src="/src/img/Icons/ScorecardIcons/No.svg" class="w1"></td><td class="pv2 ph3 v-mid city-column"><img src="/src/img/Icons/ScorecardIcons/Yes.svg" class="w1"></td><td class="pv2 ph3 v-mid city-column"><img src="/src/img/Icons/ScorecardIcons/No.svg" class="w1"></td><td class="pv2 ph3 v-mid city-column"><img src="/src/img/Icons/ScorecardIcons/Yes.svg" class="w1"></td><td class="pv2 ph3 v-mid city-column"><img src="/src/img/Icons/ScorecardIcons/DataNotAvailable.svg" class="w1"></td></tr>
`

> Uncaught (in promise) {isFromCancelledTransition: true}
(anonymous) @ cdn.min.js:5
(anonymous) @ cdn.min.js:5
(anonymous) @ cdn.min.js:5
Le @ cdn.min.js:5
out @ cdn.min.js:5
(anonymous) @ cdn.min.js:5
window.Element._x_toggleAndCascadeWithTransitions @ cdn.min.js:5
(anonymous) @ cdn.min.js:5
(anonymous) @ cdn.min.js:5
(anonymous) @ cdn.min.js:5
Ne @ cdn.min.js:5
(anonymous) @ cdn.min.js:5
lr @ cdn.min.js:1
(anonymous) @ cdn.min.js:5
r @ cdn.min.js:5
Sn @ cdn.min.js:1

So...it's just an annoyance in the console, because it works fine. But do you see a way I can modify the transition so it doesn't error?

**Comment by ekwoka** [comment]
Well, share code.

Make sure you're not running Alpine multiple times.
